src/render.js

```diff
@@ -0,0 +1,10 @@
+export const RenderPosition = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend',
+};
+
+export const renderTemplate = (container, template, place) => {
```

Comment:
название модуля/файла render а основная функция renderTemplate. Можно однообразия добавить? пусть будет файл называться RenderPosition или функция render

```suggestion
export const renderTemplate = (container, template, place = RenderPosition. BEFORE_END) => {
```

И для place можно задать значение по умолчанию BEFORE_END и не передавать этот параметр, когда это значение нужно использовать. Это уменьшит код, и обезопасит, так как иногда можно и пропустить передачу данного параметра